chart/templates/platform-rbac.yaml

```diff
@@ -0,0 +1,31 @@
+{{- if include "vcluster.rbac.createPlatformSecretRole" . }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: vc-{{ .Release.Name }}-platform-secret-role
```

Comment:
This also needs to include release namespace and a way to make sure this is not too long